_data/en/settings.yml

```diff
@@ -83,6 +84,7 @@ contact_page:
       'Pay.gov': Pay.gov
       'PBGC-MyPBA': 'PBGC - MyPBA'
       'Peace Corps': Peace Corps
+      'Postal Service Health Benefits (PSHB)': Postal Service Health Benefits (PSHB)
```

Comment:
If we could add the translations for PSHB we can merge that easily. We can double check the status of the translations for FRB.